catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/PacketBuffer.java

```diff
@@ -198,6 +206,9 @@ public long getByteCount() {
      * @param rawPacket may be null or empty
      */
     public void write(byte[] rawPacket) {
+        if (LOGGER.isTraceEnabled()) {
+            LOGGER.trace("received data packet: data={}", bytesToHex(rawPacket));
+        }
```

Comment:
This is probably overkill - it will completely fill the logs with hex data - which in operations won't be very useful (i.e., we won't be looking at the actual bytes of the packet to figure out what is happening). What might be more useful is a set of metrics that you manage and then dump periodically. For example, just increment a 'packetReceived' counter here and maybe 'totalBytesReceived'. Then at a higher level, every 5 or 10 seconds dump out a message indicating the packets/bytes received - that will provide useful information without totally overwhelming the logs.